stopcovid/dialog/engine.py

```diff
@@ -159,8 +157,8 @@ def execute(
             self._dashboard_requested,
             self._handle_opt_out,
             self._handle_opt_back_in,
-            self._validate_registration,
             self._check_response,
+            self._validate_registration,
```

Comment:
We now will handle responses to a drill if it exists before we check to see if the user has validated